src/cirq_iqm/iqm_sampler.py

```diff
@@ -25,9 +25,17 @@
 from iqm_client.iqm_client import IQMClient
 import numpy as np
 
-from cirq_iqm import IQMDevice
+from cirq_iqm.devices.iqm_device import IQMDevice, IQMDeviceMetadata
 from cirq_iqm.iqm_operation_mapping import map_operation
 
+# Mapping from IQM operation names to cirq operations
+_IQM_CIRQ_OP_MAP = {
+    'phased_rx': cirq.ops.PhasedXPowGate,
```

Comment:
There is the comment:
            "# XPow and YPow kept for convenience, Cirq does not know how to decompose them into PhasedX
            # so we would have to add those rules..."
If that one is still true we should map `phased_rx` to `PhasedXPowGate`, `XPowGate` and `YPowGate`.

Reply:
Looks like that is still true, so I added `XPowGate` and `YPowGate` also here now.